### PR TITLE
Fix messy files on Ashell

### PR DESCRIPTION
**Pull request by jimmy-huang**

### CODE REVIEW

File: src/zjs_fs.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "module": "fs",
     "require": "fs",
+    "depends": ["buffer"],
```
Comment (brianjjones) [inline src/zjs_fs.json]: Why is this being added?  I don't see buffer used in the sample.  Otherwise looks good.

Comment (jimmy-huang) [reply src/zjs_fs.json]: The reason is the the C module actually uses the zjs_buffer apis, although the JS never uses the buffer object directly.  If you build a sample without explicitly importing buffer, then you would get a complication error.

Comment (brianjjones) [reply src/zjs_fs.json]: Ah got it, thanks for the clarification.  +1

Comment (grgustaf) [reply src/zjs_fs.json]: We ought to figure out a way to have the C portion get included without the JS API overhead in these cases.

**Pull request merged**